lib/std/process.zig

```diff
@@ -1565,13 +1565,13 @@ pub fn totalSystemMemory() TotalSystemMemoryError!usize {
             if (rc != .SUCCESS) {
                 return error.UnknownTotalSystemMemory;
             }
-            return @as(usize, sbi.NumberOfPhysicalPages) * sbi.PageSize;
+            return @as(u64, sbi.NumberOfPhysicalPages) * sbi.PageSize;
         },
         else => return error.UnknownTotalSystemMemory,
     }
 }
 
-fn totalSystemMemoryLinux() !usize {
+fn totalSystemMemoryLinux() !u64 {
```

Comment:
I think the `parseInt()` for `kilobytes` on L1586 should also use `u64`.  (Otherwise 4tb host would overflow a 32-bit kb counter.)

Reply:
Good catch, didn't notice that one heh.